examples/fig10_candlestick.nim

```diff
@@ -0,0 +1,22 @@
+import plotly
+import chroma
+import random
+import sequtils
+import future
+
+var d = Trace[float32](mode: PlotMode.Lines, `type`: PlotType.Candlestick)
+# Weird cast to float32 needed because on Windows 10 the system float is float64
+d.xs = lc[float32(x) | (x <- @[1.0, 2.0, 3.0, 4.0, 5.0], float32(x) > 0), float32]
```

Comment:
I think you coul just do `d.xs = [1'f32, 2, 3, 4, 5]`
to avoid casting each, no?

Reply:
I'm still picking up Nim as I go. That quote sigil seems cool, I haven't come across it yet. I'll try.

Reply:
me too. I happen to have seen that one elsewhere.

Reply:
Yes, if you define an array or a seq, you can simply annotate the first element with the type you'd like to have and the others will be cast to that type.

Just as an aside: the list comprehension macro might disappear at some point (there was talk between @Araq and I think @krux02 about that iirc on the IRC channel).
And if you have a list of e.g. int you wish to cast to float, mapIt from sequtils is pretty useful
```nim
let x_f32 = mapIt(x_int, it.float32)
```
would do the job. :)

Reply:
Hey Vindaar thanks for the comment, I had tried many different iterations of mapIt and I could not get it to work. When I'm back at my dev environment I'll try the both approaches again and report back.